src/main/java/ch/njol/skript/effects/EffLidState.java

```diff
@@ -0,0 +1,62 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.Lidded;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Open/Close Lid")
+@Description("Open or close the lid of blocks.")
+@Examples({
+	"open the lid of {_chest}",
+	"close the lid of {_blocks::*}"
+})
+@Since("INSERT VERSION")
+public class EffLidState extends Effect {
+
+	static {
+		Skript.registerEffect(EffLidState.class,
+			"open [the] lid (of|for) %blocks%",
+			"close [the] lid (of|for) %blocks%"
```

Comment:
I reckon these patterns could be merged, and then you could have a second pattern for `%blocks%'[s] lid`. Again, logic would have to be changed to accommodate for that